CardsCashCasino/Manager/BettingManager.cs

```diff
@@ -0,0 +1,180 @@
+            _userCashValueIndicator.SetCorner(5, 5);
+        }
+
+        /// <summary>
+        /// Update method for the ChipManager.
+        /// </summary>
+        public void Update()
+        {
+            _userCashValueIndicator!.Update(500);
+        }
+
+        /// <summary>
+        /// Draw method for the ChipManager.
+        /// </summary>
+        /// <param name="spriteBatch">Holds the logic that draws the poker chips.</param>
+        public void Draw(SpriteBatch spriteBatch)
+        {
+            _userCashValueIndicator!.Draw(spriteBatch);
+        }
+
+        /// <summary>
+        /// Bet a certain amount.
+        /// </summary>
+        public void Bet(int value)
+        {
+            UserCashValue -= value;
+        }
+
+        /// <summary>
+        /// Payout a value won.
+        /// </summary>
+        public void Payout(int value)
+        {
+            UserCashValue += value;
+        }
+    }
+
+    /// <summary>
+    /// Textures for betting. Includes the menu for betting, as well as the chip pile textures.
+    /// </summary>
+    public static class BettingTextures
+    {
+        /// <summary>
+        /// LoadContent for the chip pile textures.
+        /// </summary>
+        public static void LoadContent(ContentManager content)
+        {
+            // intentionally blank
+        }
+    }
+
+    public class CashValueIndicator
+    {
+        /// <summary>
+        /// The digits representing the number.
+        /// </summary>
+        private List<IndicatorDigit> _digits = new();
+
+        /// <summary>
+        /// The dollar sign.
+        /// </summary>
+        private IndicatorCharacter _dollarSign;
+
+        /// <summary>
+        /// The previous value.
+        /// </summary>
+        private int _previousValue = 0;
+
+        /// <summary>
+        /// The top left corner of the display.
+        /// </summary>
+        private Point? _topLeftCorner;
+
+        public CashValueIndicator()
+        {
+            _dollarSign = new(DisplayIndicatorTextures.DollarSignTexture!);
+        }
+
+        /// <summary>
+        /// Updates the display with a new numerical value.
+        /// </summary>
+        /// <param name="value">The value.</param>
+        public void Update(int value)
+        {
+            if (value == _previousValue)
+                return;
+
+            _digits.Clear(); // remove the old digits
+
+            // get each digit
+            while (value > 0)
+            {
+                int digit = value % 10;
+                value /= 10;
+
+                IndicatorDigit indicatorDigit = new IndicatorDigit();
+                indicatorDigit.Update(digit);
+                _digits.Add(indicatorDigit);
+            }
+
+            _digits.Reverse(); // put them in the correct order
+
+            if (_topLeftCorner is not null)
+                CalculateDigitPositions(((Point)_topLeftCorner).X + 21, ((Point)_topLeftCorner).Y);
```

Comment:
Wait I'm confused, where is the constant `21` coming from here?

Reply:
its the width of a digit object. ill move it to constants.

Reply:
going to resolve in my next ticket so this can get merged.